prime-router/src/main/kotlin/transport/RESTTransport.kt

```diff
@@ -111,8 +111,11 @@ class RESTTransport(private val httpClient: HttpClient? = null) : ITransport {
                         val httpHeaders = getHeaders(restTransportInfo, reportId)
                         var accessToken: String? = null
 
-                        if (restTransportInfo.authType == "apiKey") {
+                        if (restTransportInfo.authType == "Api Key") {
                             val apiKeyCredential = credential as UserApiKeyCredential
+                            if (httpHeaders["shared-api-key"] == "From Vault") {
```

Comment:
Can we just create the httpHeader without checking if (httpHeaders["shared-api-key"] == "From Vault")?

Reply:
Yes, we can do that.

Reply:
I didn't get any gitleak issues when commiting. Maybe you can disable it, I'll add it as a suggestion here and see if it lets you that way.

Reply:
Configure gitleaks for allow "apiKey",